watcher/src/controllers/replayer.js

```diff
@@ -59,34 +58,45 @@ const _replayPastOnly = async ({
 };
 
 const startReplayer = async () => {
-  const agenda = await getAgenda(chainId);
-  agenda.define(
+  const queue = getQueue(EVENT_REPLAY_JOB);
+
+  // Create worker to process jobs
+  getWorker(EVENT_REPLAY_JOB, async (job) => {
+    try {
+      await _replayPastOnly({
+        handleIndexedBlock: async (blockNumber) => {
+          await setCheckpointBlock(blockNumber);
+          // Extend job lock after every iteration
+          await job.updateProgress(100);
```

Comment:
Extending the lock time was specific to agenda to avoid job timeout while they were still running, this seems unnecessary with bullmq.
Additionnaly the job progess is inaccurate.
`handleIndexedBlock` is called after each batch processed by `_replayPastOnly`, likely multiple times within the job lifetime.

```suggestion
```